xterm/index.cgi

```diff
@@ -271,11 +273,34 @@ my $term_script = <<EOF;
 	socket.onopen = function() {
 		var term = new Terminal($termjs_opts{'Options'}),
 		    attachAddon = new AttachAddon.AttachAddon(this);
+		    fitAddon = new FitAddon.FitAddon();
 		term.loadAddon(attachAddon);
+		term.loadAddon(fitAddon);
 		term.open(termcont);
+		fitAddon.fit();
 		term.focus();
+		
+		// On resize event triggered by fit()
+		term.onResize(function(e) {
+			// \033[8;(40);(100)t
+			// socket.send('\\\\033\\\\[8;('+e.rows+');('+e.cols+')t');
+			term.write('\\033[8;'+e.rows+';'+e.cols+'t')
+			socket.send(JSON.stringify(e));
```

Comment:
This seems risky ... what if the some valid JSON gets sent as part of the normal terminal operation?

Reply:
This is not a production ready code. If that's the path to make resizes work, I will make it very specific type of JSON object to work with. Now, it's just for testing purposes.